### ISSUE TITLE
Support crop + scale mode

### ISSUE THREAD

**Issue by chrisbenincasa** [chrisbenincasa/tunarr#1031]
We should support use cases for converting 16:9 -> 4:3, etc.

**Comment by ameturpoet** [comment]
Good idea! Having different channel logo placement settings for 16:9 and 4:3 would also be helpful - it's always a bit of a pet peeve for 4:3 content to see video letterboxed and pillarboxed and have the channel logo floating out in the void :P

**Comment by DarkManX16** [comment]
Hopefully this gets added to an update soon

**Comment by liamblink182** [comment]
I'd also find this useful. I run a Tunarr setup pulling from a Plex library, and within a single channel, individual shows/movies often have genuinely different native aspect ratios from each other — some 4:3, some standard widescreen, some wider cinematic ratios. With only pad/letterbox currently available, this means the same channel looks inconsistent depending on which title happens to be playing: some titles fill the frame cleanly, others get bars.

It also compounds with channel watermarks/logos — like ameturpoet mentioned above, when a title gets letterboxed the logo ends up sitting in what's now empty padding space rather than over the actual picture, which looks off compared to titles that don't need padding.

A per-channel (or even per-source) toggle between the current pad/letterbox behavior and a crop-to-fill mode would cover both cases well — pad where preserving the full picture matters most, crop where a consistent, gap-free frame matters more. Would be a great addition.
